templates/download/alternative-downloads.html

```diff
@@ -6,137 +6,236 @@
-  <div class="row" id="network-installer">
-    <div class="col-8">
-      <h2>Network installer</h2>
-      <p>The network installer lets you install Ubuntu over a network. It includes the minimal set of packages needed to
-        start and the rest of the packages are downloaded over the network. Since only current packages are downloaded,
-        there is no need to upgrade packages immediately after installation.</p>
-      <p>The network installer is ideal if you have a computer that cannot run the graphical installer, for example,
-        because it does not meet the minimum requirements for the live CD/DVD, or because the computer requires extra
-        configuration before a graphical desktop can be used. The network installer is also useful if you want to
-        install Ubuntu on a large number of computers at once.</p>
-      <p>For 22.04 LTS, users can use the new Ubuntu Live installer to setup and configure a network install.</p>
-      <ul class="p-list">
-        <li class="p-list__item is-ticked"><a
-            href="https://discourse.ubuntu.com/t/netbooting-the-live-server-installer">Instructions for the {{
-            releases.lts.short_version }} LTS Ubuntu Live installer</a></li>
-      </ul>
+<div class="p-section">
+  <div class="row">
+    <div class="col-3">
+      <div class="p-side-navigation--raw-html is-sticky" id="drawer" style="top: 80px;">
+        <button class="p-side-navigation__toggle js-drawer-toggle" aria-controls="drawer">
+          Toggle table of contents
+        </button>
+        <div class="p-side-navigation__overlay js-drawer-toggle" aria-controls="drawer"></div>
+        <nav class="p-side-navigation__drawer" aria-label="documentation side navigation">
+          <div class="p-side-navigation__drawer-header">
+            <button class="p-side-navigation__toggle--in-drawer js-drawer-toggle" aria-controls="drawer">
+              Toggle table of contents
+            </button>
+          </div>
+          <ul>
+            <li class="p-side-navigation__item">
+              <a class="highlight-link is-active" href="#bit-torrent">BitTorrent</a>
+            </li>
+            <li class="p-side-navigation__item">
+              <a class="highlight-link" href="#other-images-and-mirrors">Other images and mirrors</a>
+            </li>
+            <li class="p-side-navigation__item">
+              <a class="highlight-link" href="#past-releases-and-other-flavours">Past releases and other flavours</a>
+            </li>
+            <li class="p-side-navigation__item">
+              <a class="highlight-link" href="#network-installer">Network installer</a>
+            </li>
+          </ul>
+        </nav>
+      </div>
     </div>
-  </div>
-</div>
+    <div class="col-9">
+      <div class="p-section">
+        <hr class="p-rule"/>
+        <div class="p-section--shallow">
+          <h2 id="bit-torrent" class="question-heading" style="scroll-margin-top: 3.4rem;">BitTorrent</h2>
```

Comment:
Might be good to put the scroll styles in a scss file so it can be reused if this kind of layout is requested again

Reply:
It varies page to page depending on the layout and such